src/framework/main_control_loop.py

```diff
@@ -11,6 +12,7 @@ def setup(self):
         self.read_camera = ReadCamera(self.config, self.sfr)
         self.point_tracker = PointTracker(self.config, self.sfr)
         self.mission_manager = MissionManager(self.config, self.sfr)
+        #self.time_manager = TimeManager(self.config, self.sfr)
```

Comment:
Uncomment this and add time manager to the default, setup, and execute function calls in MCL's body
@ezh20